packages/dx-react-grid-material-ui/src/templates/toolbar/toolbar.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { Toolbar as ToolbarMUI } from 'material-ui';
+import { withStyles } from 'material-ui/styles';
+import { darken, fade, lighten } from 'material-ui/styles/colorManipulator';
+
+const styles = theme => ({
+  toolbar: {
+    paddingTop: theme.spacing.unit * 1.5,
+    borderBottom: `1px solid ${
+      theme.palette.type === 'light'
+        ? lighten(fade(theme.palette.text.lightDivider, 1), 0.925)
+        : darken(fade(theme.palette.text.lightDivider, 1), 0.685)
```

Comment:
What are these calculations with "magic numbers" for? Is not there a builtin color that suits our needs?

Reply:
It's from [MUI](https://github.com/mui-org/material-ui/blob/v1-beta/src/Table/TableCell.js) repo. I can't find any builtin colors for this.